### PR TITLE
3.1.0

### PR DESCRIPTION
**Pull request by fmina**
@helenol @burrimi 
Sync mav_msgs with the devel repo.

### CODE REVIEW

File: mav_msgs/include/mav_msgs/eigen_mav_msgs.h [hunk 1]
```diff
@@ -186,6 +231,9 @@ struct EigenOdometry {
   inline double getYaw() const {
     return yawFromQuaternion(orientation_W_B);
   }
+  inline void getEulerAngles(Eigen::Vector3d& euler_angles) const {
```
Comment (burrimi) [inline mav_msgs/include/mav_msgs/eigen_mav_msgs.h]: `inline void getEulerAngles(Eigen::Vector3d* euler_angles) const { `

File: mav_msgs/include/mav_msgs/eigen_mav_msgs.h [hunk 2]
```diff
@@ -186,6 +231,9 @@ struct EigenOdometry {
...
+    getEulerAnglesFromQuaternion(orientation_W_B, &euler_angles);
```
Comment (burrimi) [inline mav_msgs/include/mav_msgs/eigen_mav_msgs.h]: `getEulerAnglesFromQuaternion(orientation_W_B, euler_angles);`

**Comment by helenol** [comment]
Thanks! Medium-term plan (before end of the year) is to fully merge mav_comm_devel into mav_comm and delete mav_comm_devel, FYI.

Agree with Burri's comments, fine from my side otherwise.

File: mav_msgs/include/mav_msgs/eigen_mav_msgs.h (hunk 1 above)
Comment (fmina) [inline mav_msgs/include/mav_msgs/eigen_mav_msgs.h]: done

**Pull request merged**